### PR TITLE
Let Share && AddToMyList to be deactivated at will for the Discipline Template

### PR DESCRIPTION
**Pull request by StefanoA1**
**Detailed purpose of the PR**

**Result and observation**
Kapture 2021-07-23 at 11 46 34
Kapture 2021-07-23 at 11 51 02
Screenshot 2021-07-23 at 16 12 00

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by StefanoA1** [comment]
[rendre désactivable le bloc share & add to my list  sur la page discipline](https://trello.com/c/R0Eqs4TY/221-rendre-d%C3%A9sactivable-le-bloc-share-add-to-my-list-sur-la-page-discipline)

**Review by adamska27 — APPROVED** [review]

File: packages/@coorpacademy-components/src/template/common/discipline/index.js [hunk 1]
```diff
@@ -44,85 +46,117 @@ const Discipline = (props, context) => {
...
-                <DisciplineCTA
-                  type={'discipline'}
-                  start={start}
-                  buy={buy}
-                  startLabel={startLabel}
-                  buyLabel={buyLabel}
-                />
+  const buildAddToMyList = options => (
+    <AddToMyList
+      style={options.isTablet ? style.addToMyListBtnTablet : style.addToMyListBtn}
```
Comment (silouone) [inline packages/@coorpacademy-components/src/template/common/discipline/index.js:51]: on peut pas gerer ca cote css avec le juste la detection @media mobile @media desktoo?

Comment (StefanoA1) [reply packages/@coorpacademy-components/src/template/common/discipline/index.js:51]: oui c'est vrai, je vais le faire 👍

File: packages/@coorpacademy-components/src/template/common/discipline/index.js [hunk 2]
```diff
@@ -44,85 +46,117 @@ const Discipline = (props, context) => {
...
-                  <AddToMyList
-                    style={style.addToMyListBtn}
-                    addToMyListButton={addToMyListButton}
-                    favorite={favorite}
-                    onFavoriteClick={onFavoriteClick}
-                  />
-                </div>
+  const buildShare = options => (
+    <Share
+      style={options.isTablet ? style.shareBtnTablet : style.shareBtn}
```
Comment (silouone) [inline packages/@coorpacademy-components/src/template/common/discipline/index.js:60]: idem

File: packages/@coorpacademy-components/src/template/common/discipline/index.js [hunk 3]
```diff
@@ -44,85 +46,117 @@ const Discipline = (props, context) => {
...
+    <div data-name="discipline" className={style.container} key="discipline">
+      <div className={style.leftSection}>
+        {disciplineHeader}
+        <div className={style.mobileAuthorCtaSection}>
+          <div className={style.columnReverse}>
+            <div className={style.cta}>{disciplineCTA}</div>
+            {disableAddToMyList
+              ? null
+              : buildAddToMyList({
+                  isTablet: true
```
Comment (silouone) [inline packages/@coorpacademy-components/src/template/common/discipline/index.js:98]: si ca marche as expected cote css , du coup tu pourra clean ca

File: packages/@coorpacademy-components/src/template/common/discipline/style.css [hunk 4]
```diff
@@ -66,32 +66,49 @@
...
 
-.addToMyListBtn {
-  flex: 1.3;
-  margin-left: 15px;
-}
-
-.addToMyListBtnTablet {
-  margin-top: 15px;
-}
+@media wide {
```
Comment (silouone) [inline packages/@coorpacademy-components/src/template/common/discipline/style.css:75]: à tester en deploy, on en a pas encore ds la stack component, on a que mobile, tablet et desktop

**Review by silouone — APPROVED** [review]

**Pull request merged**